src/app/components/apps-view/apps-view.component.html

```diff
@@ -41,20 +38,9 @@
     </mat-form-field>
   </div>
   <mat-form-field class="search-wrapper white-mat-form-field">
-    <input
```

Comment:
Pls use prettier in your IDE so that it honors prettierrc config file and does not do unnecessary file formatting as part of the PR changes.

Reply:
reverted